stdlib/array.gr

```diff
@@ -105,7 +104,7 @@ export let init /*: (Number, Number -> a) -> Array<a>*/ =
   let array = allocateArray(length)
   let mut index = 0n
   for (let mut i = 0n; WasmI32.ltS(i, byteLength); i = WasmI32.add(i, 4n)) {
-    // [FIXME] This line fails the array/map test suite (#815)
+    // TODO(#815): This line fails the array/map test suite
```

Comment:
This issue has been closed but when I remove the uncomment the line the tests fail i dont know if this is an issue with the tests being wrong, or this being an outdated comment, or if another issue needs to be open for this

Reply:
I think this just changed since the original implementation. You can delete the FIXME and the assertion (since it's no longer testing what it once did).